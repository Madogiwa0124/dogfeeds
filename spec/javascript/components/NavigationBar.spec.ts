import { mount } from "@vue/test-utils";
import Component from "@js/components/NavigationBar.vue";

describe("components/NavigationBar.vue", () => {
  it("snapshot", () => {
    const wrapper = mount(Component);
    expect(wrapper.element).toMatchSnapshot();
  });
});
