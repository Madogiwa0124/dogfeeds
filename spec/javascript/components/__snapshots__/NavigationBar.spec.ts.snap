// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/NavigationBar.vue snapshot 1`] = `
<nav
  aria-label="main navigation"
  class="navbar"
  role="navigation"
>
  <div
    class="navbar-brand"
  >
    <a
      class="navbar-item"
      href="/"
    >
      <img
        alt="favicon"
        height="28"
        src="/favicon.ico"
        width="28"
      />
       Dogfeeds 
    </a>
    <span
      class="navbar-item tag-line"
    >
      いろんなRSSをまとめて、1つのRSSに。
    </span>
    <a
      aria-expanded="false"
      aria-label="menu"
      class="navbar-burger burger"
      data-target="header-navbar"
      role="button"
    >
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
    </a>
  </div>
  <div
    class="navbar-menu"
    id="header-navbar"
  >
    <div
      class="navbar-end"
    >
      <div
        class="navbar-item"
      >
        <a
          class="button is-primary is-rounded"
          href="/feeds/new"
        >
          RSSフィードを登録する
        </a>
      </div>
      <div
        class="navbar-item"
      >
        <a
          class="button is-info is-rounded"
          href="/feeds"
        >
          RSSフィードを見る
        </a>
      </div>
      <div
        class="navbar-item has-dropdown is-hoverable"
      >
        <a
          class="navbar-link is-arrowless"
        >
           MENU   
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-bars fa-w-14"
            data-icon="bars"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class=""
              d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
              fill="currentColor"
            />
          </svg>
        </a>
        <div
          class="navbar-dropdown is-right"
        >
          <a
            class="navbar-item"
            href="/boards"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-boxes fa-w-18"
              data-icon="boxes"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                class=""
                d="M560 288h-80v96l-32-21.3-32 21.3v-96h-80c-8.8 0-16 7.2-16 16v192c0 8.8 7.2 16 16 16h224c8.8 0 16-7.2 16-16V304c0-8.8-7.2-16-16-16zm-384-64h224c8.8 0 16-7.2 16-16V16c0-8.8-7.2-16-16-16h-80v96l-32-21.3L256 96V0h-80c-8.8 0-16 7.2-16 16v192c0 8.8 7.2 16 16 16zm64 64h-80v96l-32-21.3L96 384v-96H16c-8.8 0-16 7.2-16 16v192c0 8.8 7.2 16 16 16h224c8.8 0 16-7.2 16-16V304c0-8.8-7.2-16-16-16z"
                fill="currentColor"
              />
            </svg>
             みんながまとめたRSSフィードを見る 
          </a>
          <hr
            class="navbar-divider"
          />
          <a
            class="navbar-item"
            href="/boards/new"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-box fa-w-16"
              data-icon="box"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                class=""
                d="M509.5 184.6L458.9 32.8C452.4 13.2 434.1 0 413.4 0H272v192h238.7c-.4-2.5-.4-5-1.2-7.4zM240 0H98.6c-20.7 0-39 13.2-45.5 32.8L2.5 184.6c-.8 2.4-.8 4.9-1.2 7.4H240V0zM0 224v240c0 26.5 21.5 48 48 48h416c26.5 0 48-21.5 48-48V224H0z"
                fill="currentColor"
              />
            </svg>
              RSSフィードをまとめる 
          </a>
          <hr
            class="navbar-divider"
          />
          <a
            class="navbar-item"
            href="/mypage"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-user fa-w-14"
              data-icon="user"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                class=""
                d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                fill="currentColor"
              />
            </svg>
              マイページ 
          </a>
        </div>
      </div>
    </div>
  </div>
</nav>
`;
